src/Symfony/Component/VarDumper/Tests/Caster/SplCasterTest.php

```diff
@@ -159,7 +159,7 @@ public function testCastObjectStorageIsntModified()
     public function testCastObjectStorageDumpsInfo()
     {
         $var = new \SplObjectStorage();
-        $var->attach(new \stdClass(), new \DateTime());
+        $var->attach(new \stdClass(), new \DateTimeImmutable());
 
         $this->assertDumpMatchesFormat('%ADateTime%A', $var);
```

Comment:
should be updated to look for `DateTimeImmutable` and not `DateTime`

Reply:
Fixed